content/sensu-go/5.4/reference/checks.md

```diff
@@ -93,7 +72,221 @@ At every execution of a check command – regardless of success or failure – t
 Sensu agent publishes the check’s result for eventual handling by the **event
 processor** (the Sensu backend).
 
-### Check token substitution
+## Check scheduling
+
+Checks are scheduled by the Sensu backend, which
+publishes check execution requests to entities via a [publish-subscribe
+model][2].
+
+### Subscriptions
+
+Checks have a defined set of subscriptions, transport
```

Comment:
Should this be `subscriptions and transport topics`? Or is `transport topics` another way of saying `subscriptions`?

Reply:
I'd say these concepts are the same, or that transport topics are an implementation detail of subscriptions. Can probably simply avoid mentioning them.

Subscriptions provide a mechanism for creating a many-to-many mapping between checks and entities.